### PR TITLE
feat: Apply change pin

### PR DESCRIPTION
**Pull request by k3255**
## Description
added ability to change pin

## Related Issue (Optional)
N/A

## Changes
added ability to change pin

## Screenshots (Optional)
N/A

## Additional Comments (Optional)
N/A

### CODE REVIEW

File: source/did-ca-aos/app/src/main/java/org/omnione/did/ca/config/Constants.java [hunk 1]
```diff
@@ -42,12 +42,13 @@ public class Constants {
...
-    public final static String PIN_INPUT_TEXT = "Please input a Pin";
-    public final static String PIN_REGISTER_LOCK_TEXT = "Please register a Lock/Unlock Pin";
-    public final static String PIN_INPUT_LOCK_TEXT = "Please input a Lock/Unlock Pin";
-    public final static String PIN_NOT_MATCH_TEXT = "Pin number does not match!!!!!!!!!";
-
+    public final static String PIN_REGISTER_TEXT = "Please register a PIN";
+    public final static String PIN_INPUT_TEXT = "Please input a PIN";
+    public final static String PIN_REGISTER_LOCK_TEXT = "Please register a Unlock PIN";
+    public final static String PIN_INPUT_LOCK_TEXT = "Please input a Unlock PIN";
+    public final static String PIN_NOT_MATCH_TEXT = "PIN number does not match!!!!!!!!!";
```
Comment (djpark0402) [inline source/did-ca-aos/app/src/main/java/org/omnione/did/ca/config/Constants.java:49]: The syntax is different on iOS(PIN does not match)

File: source/did-ca-aos/app/src/main/java/org/omnione/did/ca/ui/PinActivity.java [hunk 2]
```diff
@@ -240,21 +251,114 @@ private void authenticatePin(){
...
+            message.setText(Constants.PIN_CHANGE_TEXT);
+
             for (int i = 0; i <= 5; i++) {
                 pw[i].setImageDrawable(ContextCompat.getDrawable(PinActivity.this, R.drawable.omni_pin_num_in));
             }
+            tempPassword = password;
             password = "";
-            finish();
+            isChange = true;
+            isResister = true;
```
Comment (djpark0402) [inline source/did-ca-aos/app/src/main/java/org/omnione/did/ca/ui/PinActivity.java:307]: isResister -> isRegister

File: source/did-ca-aos/app/src/main/java/org/omnione/did/ca/ui/SettingsExpandableActivity.java [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+        childData.add(childListA);
+
+        ArrayList<HashMap<String, String>> childListB = new ArrayList<>();
+
+        HashMap<String, String> childBA = new HashMap<>();
+        childBA.put("data", "Setting up a fignerprint for Signing");
+        childListB.add(childBA);
+
+        HashMap<String, String> childBB = new HashMap<>();
+        childBB.put("data", "Setting up a fignerprint for Unlock");
```
Comment (djpark0402) [inline source/did-ca-aos/app/src/main/java/org/omnione/did/ca/ui/SettingsExpandableActivity.java:90]: fignerprint -> fingerprint

**Review by djpark0402 — CHANGES_REQUESTED** [review]

**Comment by k3255** [comment]
sorry. I will edit this part and apply it right away.

**Review by djpark0402 — APPROVED** [review]
perfect!!

**Pull request merged**